charts/weave-gitops/values.yaml

```diff
@@ -0,0 +1,4 @@
+namespace: flux-system
+image: v0.6.2
+containerRegistry: ghcr.io/weaveworks/wego-app
```

Comment:
I think we should use `gitops-server`, but note you can't actually install gitops properly outside of the tilt/dev env yet (#1482), because we haven't been publishing images we can use for that, which… Well, after this it'll be easier to do without interfering with main, at least.